autoload/ollama/edit.vim

```diff
@@ -5,10 +5,19 @@ let s:buf = -1
 " avoid starting a 2nd edit job while one is in progress
 let g:edit_in_progress = 0
 
+" Function to check if embedded Python is available
+function! ollama#edit#HasEmbeddedPython() abort
+    return exists('g:ollama_embedded_python') && g:ollama_embedded_python != 0
```

Comment:
*minor*

In `autoload/ollama/setup.vim` only the value of `g:ollama_embedded_python` is checked:

```vim
            if g:ollama_embedded_python
              call s:SetupPyVEnv()
            endif
```

Here both existence and value are checked.  While checking existence is more robust, if the assumption is that this value will always be set by the plugin setup code, the existence check becomes redundant.

In any case, it seems inconsistent to treat this variable one way in one location and another way in another.
I think either both locations should check existence, or neither should.
I would probably recommend the latter.
It would make the check simpler.